frontend/src/components/card/HostActionCard.tsx

```diff
@@ -1,25 +1,43 @@
 import React from 'react';
 import styled from 'styled-components';
-import { SmallActionText } from '../core/Text';
+import { SmallActionHeaderText, SmallActionText } from '../core/Text';
 import { User } from '../../api/User';
 
-const Content = styled.div`
+type PlayerIconType = {
+  isActive: boolean,
+};
+
+const Content = styled.div<PlayerIconType>`
   // Center div above parent
   position: absolute;
```

Comment:
if absolute doesn't work (in terms of getting it to overflow outside of the container), how about position fixed? If not yeah I think it'd be good to address this in some PR later since it can be slightly difficult to see the card when the player is near the bottom of the container (or maybe if we increased the height of the container?)

Reply:
Based on some quick testing that first solution may work - how do you get the fixed component to center right above the card? When I added that it put it relative to the page, but adding `position: relative` to the outer container didn't help.

Reply:
I'm not sure how to do this - based on this [description of position fixed](https://stackoverflow.com/questions/5209814/can-i-position-an-element-fixed-relative-to-parent) and some initial attempts, I don't know how this would work.

I'm open to investigating more (maybe pair programming?), though I think that might be a best for a separate PR since there are additional last few updates that would be helpful (like setting up flex on this page so the bottom containers go to the end of the page, and no farther).

Reply:
Yeah that sounds good - I'll make a list at the bottom for this and the others mentioned so that it's consolidated in one place